xbmc/addons/binary/interfaces/api1/Peripheral/AddonCallbacksPeripheral.cpp

```diff
@@ -37,7 +37,7 @@ namespace Peripheral
 {
 
 CAddonCallbacksPeripheral::CAddonCallbacksPeripheral(ADDON::CAddon* addon)
-  : ADDON::IAddonInterface(addon, APILevel(), Version()),
+  : ADDON::IAddonInterface(addon, 1, "0.0.1"),
```

Comment:
Thanks, has not looked to the kodi_peripheral_types.h only the libKODI_peripheral.h

Becomes changed.